.buildkite/pipeline.yml

```diff
@@ -141,6 +144,9 @@ steps:
       bundle exec fastlane distribute_release_build
     agents:
       queue: "mac" # 
+    concurrency_group: 'studio/release-manifest-update'
+    concurrency: 1
+    concurrency_method: eager
```

Comment:
```suggestion
    # Using concurrency_group to ensure the CI builds from `trunk` & the git tag, which are likely to run at roughly the same time,
    # don't run into a race condition when downloading+updating+uploading the `releases.json` file from/to S3
    concurrency_group: 'studio/release-manifest-update'
    concurrency: 1
    concurrency_method: eager
```

Reply:
Added in https://github.com/Automattic/studio/pull/183/commits/5d2ff0843de4ea9ac4793b827bb2bbc86166191d